src/displayapp/fonts/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(FONTS jetbrains_mono_42 jetbrains_mono_76 jetbrains_mono_bold_20
+   jetbrains_mono_extrabold_compressed lv_font_navi_80 lv_font_sys_48
+   open_sans_light)
+find_program(LV_FONT_CONV "lv_font_conv" NO_CACHE REQUIRED)
+configure_file(${CMAKE_CURRENT_LIST_DIR}/jetbrains_mono_bold_20.c_zero.patch
+   displayapp/fonts/jetbrains_mono_bold_20.c_zero.patch COPYONLY)
+foreach(FONT ${FONTS})
+   set_source_files_properties(displayapp/fonts/${FONT}.c
+      PROPERTIES GENERATED TRUE)
+
+   add_custom_command(OUTPUT displayapp/fonts/${FONT}.c
+      COMMAND python ${CMAKE_CURRENT_LIST_DIR}/generate.py
```

Comment:
```suggestion
      COMMAND python3 ${CMAKE_CURRENT_LIST_DIR}/generate.py
```

Reply:
the typing should be solved by using python3 instead of 2, but this will break some newer linux builds - which don't have a "python3" executable anymore.
Also, in old enough distributions perhaps the python3 is not new enough to have f-strings,

so Instead I'll convert things to version agnostic code.

Reply:
that worked, but ended up being a bit more of a mess then expected,
so I changed to use python3, and when it's not found to use python. (if there's only python2, it will find it - and fail when running.)

Reply:
Or maybe we should update the version of the base image of the docker (to a version that uses python3 by default)? 18.04 is probably quite old.

Reply:
changed to use ubuntu 20.04 (the current LTS), which does indeed solve the issue without any of the other parts (which I removed)